src/com/minseung/datastructure/StackAndQueue.java

```diff
@@ -0,0 +1,75 @@
+package com.minseung.datastructure;
```

Comment:
I just realize this; since there is a website with this qualifier, we need to use different package name.